### PR TITLE
Added check for Frontend Ingress

### PR DESCRIPTION
**Pull request by mrsabath**
Added check for Frontend Ingress and instructions for starting port forwarding when no Ingress configured

### CODE REVIEW

File: charts/tornjak/templates/NOTES.txt [hunk 1]
```diff
@@ -74,8 +74,22 @@ Universal Trusted Workload Identity Service has completed.
       }
 {{- end }}
 
-Tornjak Backend API: {{ include "tornjak.apiURL" . }}
+{{- if not .Values.tornjak.config.backend.ingress }}
 
+No Backend Ingress provided. 
+Use port forwarding to access the Backend port:
+      kubectl -n spire port-forward spire-server-0 10000:10000
```
Comment (maia-iyer) [inline charts/tornjak/templates/NOTES.txt:81]: it'd be nice if the port number could be provided in a variable perhaps?

**Review by maia-iyer — APPROVED** [review]
LGTM minor nits

File: charts/tornjak/templates/NOTES.txt (hunk 1 above)
Comment (mrsabath) [reply charts/tornjak/templates/NOTES.txt:81]: The ports are currently hardcoded in Tornjak:
https://github.com/spiffe/tornjak/blob/6eb59e1c561ae959b9bc9efe731bf48a39e08a06/tornjak-backend/cmd/agent/agent.go#L69
and also for TLS and mTLS:
https://github.com/spiffe/tornjak/blob/6eb59e1c561ae959b9bc9efe731bf48a39e08a06/run_tornjak.sh#L63-L65
So unless me allow changing them during the image instantiation, there is no point of providing them as parameters here.

File: charts/tornjak/templates/NOTES.txt [hunk 2]
```diff
@@ -74,8 +74,22 @@ Universal Trusted Workload Identity Service has completed.
...
+{{- end }}
+
+{{- if not .Values.tornjak.config.frontend.ingress }}
+
+No Frontend Ingress provided. 
+Use port forwarding to access the Frontend port:
+      kubectl -n spire port-forward spire-server-0 3000:3000
```
[the remark this answers is not in the document]
Comment (mrsabath) [reply charts/tornjak/templates/NOTES.txt:88]: I think the port-forward applies to the Pod or Deployment. I should, however make the namespace a variable

**Pull request merged**